src/pvHelpers/models/approval_group.py

```diff
@@ -0,0 +1,114 @@
+import types
```

Comment:
hmm, maybe rather than `pvHelpers/modesl` u can call the module  `pvHelpers/approval_group` so we can put all 
the generic apg related things (sharding versions, export shard versioning, ... ) in this module.